interfaces/PA-App/src/app/services/padata-types.enum.ts

```diff
@@ -1,16 +1,17 @@
 export enum PaDataTypes {
-    conversationHistory = 'conversationHistory',
-    country = 'countryId',
-    credDefId = 'credDefId',
-    credentialRequest = 'credentialRequest',
-    did = 'did',
-    didShort = 'didShort',
-    fsp = 'fsp',
-    language = 'languageCode',
-    myAnswers = 'myAnswers',
-    myAppointments = 'myAppointments',
-    myPrograms = 'myPrograms',
-    programId = 'programId',
-    timeslot = 'timeslotChoice',
-    wallet = 'wallet',
+  conversationHistory = 'conversationHistory',
```

Comment:
I think something has changed in your "auto-format-on-save"-settings...
As this change should only include adding the line: `phoneNumber = 'phoneNumber',`.